File: patterns/pattern_10.cpp
#include<iostream>
using namespace std;
int main()
{
	int n;
	cout <<"Enter a range: ";
	cin >>n;
	
	for(int i=1; i<=n; i++)
	{
		for(int j=1; j<=i; j++)
		{
			cout <<"*";
		}
		cout <<endl;
	}
	
	for(int i=1; i<=n; i++)
	{
		for(int j=1; j<=n-i; j++)
		{
			cout <<"*";
		}
		cout <<endl;
	}
	return 0;
}

File: patterns/Pattern10.java
import java.util.*;

class Pattern10{

	public static void main(String args[]){
		Scanner sc= new Scanner(System.in);

		int n;
		System.out.print("Enter a range: ");
		n= sc.nextInt();
		
		for(int i=1; i<=n; i++)
		{
			for(int j=1; j<=i; j++)
			{
				System.out.print("*");
			}
			System.out.println();
		}
		
		for(int i=1; i<=n; i++)
		{
			for(int j=1; j<=n-i; j++)
			{
				System.out.print("*");
			}
			System.out.println();
		}
	}
}